resources/styles.css

```diff
@@ -307,3 +307,7 @@ html, body {
     width: 100%;
     margin-top: 10px;
 }
+
+em{
```

Comment:
should this affect to every em tag? maybe it should have it's own class or use css nesting (`.config-menu em{` or something like that)